hack/common.sh

```diff
@@ -40,10 +41,34 @@ OSDK_WMCO_management() {
   local COMMAND=$1
   local OSDK_PATH=$2
 
-  $OSDK_PATH $COMMAND packagemanifests \
-    --olm-namespace openshift-operator-lifecycle-manager \
-    --operator-namespace openshift-windows-machine-config-operator \
-    --operator-version 0.0.0 \
+  if [[ "$COMMAND" = "run" ]]; then
+    run_wmco $OSDK_PATH
+  fi
+  if [[ "$COMMAND" = "cleanup" ]]; then
+    cleanup_wmco $OSDK_PATH
+  fi
+}
+
+run_wmco() {
```

Comment:
Having `*_wmco()` and `*_WMCO()` is really confusing. I don't see an advantage in having `*_wmco()` and would prefer the code being inline.